lib/smart_proxy_ansible/reader_helper.rb

```diff
@@ -32,7 +32,7 @@ def path_from_config(config_key)
 
         def playbook_or_role_full_name(path)
           parts = path.split('/')
-          playbook = parts.pop
+          playbook = parts.pop.gsub('.yaml', '').gsub('.yml','')
```

Comment:
I believe it would be better to use a single (g)sub with regex :)
```suggestion
          playbook = parts.pop.sub(/\.ya?ml/, '')
```